src/api/Nhs.Appointments.Persistance/BookingCosmosDocumentStore.cs

```diff
@@ -73,6 +78,15 @@ public async Task<bool> UpdateStatus(string bookingReference, string status)
         return true;
     }
 
+    public async Task SetReminderSent(Booking booking)
```

Comment:
This method only needs site and reference so I would change the signature to reflect this as it makes the method more reusable (i.e, it doesn't need the whole booking object so it shouldn't require one)